docs/api/graphql_queries.md

```diff
@@ -181,6 +181,72 @@ Response:
 }
 ```
 
+### Get Matrix Field Type
+
+To get a Matrix Field Type, use `product` field.
+
+The example below shows Product Content Type that has two fields:
+
+- `name`: `ezstring`
+- `features`: `ezmatrix` with two columns: `name` and `price`
+
+Fields of that type return a list of `ProductFeaturesRow`.
+
+```
+{
+  content {
+    product(contentId: 123) {
+      name
+      features {
+        name
+        price
+      }
+    }
+  }
+}
+```
+
+Response:
```

Comment:
This is wrong. It is not the response for the query above, it is a query for the type and field definition's details.

In regards to the field definition, the reference doc for that field type should list the settings that are exposed (minimumRows, columns (name + identifier).